docs/server/request.md

```diff
@@ -246,3 +246,35 @@ A shortcut method that calls JSON.parse on the request's body.
 ```js
 req.jsonBody();
 ```
+
+### overrideRecordingName
+
+Override the recording name for the request.
```

Comment:
nit: docs could be more descriptive.  mentioning it'll filter out of the original top level recording name and into it's own.